core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -564,6 +564,13 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
     val encryptionEnabled = get(NETWORK_ENCRYPTION_ENABLED) || get(SASL_ENCRYPTION_ENABLED)
     require(!encryptionEnabled || get(NETWORK_AUTH_ENABLED),
       s"${NETWORK_AUTH_ENABLED.key} must be enabled when enabling encryption.")
+
+    val executorTimeoutThreshold = Utils.timeStringAsSeconds(get("spark.network.timeout", "120s"))
+    val executorHeartbeatInterval = Utils.timeStringAsSeconds(
+      get("spark.executor.heartbeatInterval", "10s"))
+    require(executorHeartbeatInterval > executorTimeoutThreshold, s"The value of" +
+      s"spark.network.timeout' must be no less than the value of" +
+      s" 'spark.executor.heartbeatInterval'.")
```

Comment:
nit: The whitespace should always go to the end of the previous line instead of the beginning.

Reply:
Sorry for that,i will check more carefully!

Reply:
Done